ui/decredmaterial/collapsible.go

```diff
@@ -6,66 +6,220 @@ import (
 	"gioui.org/layout"
 	"gioui.org/unit"
 	"gioui.org/widget"
+	"gioui.org/widget/material"
 )
 
 type Collapsible struct {
-	isExpanded            bool
-	buttonWidget          *widget.Clickable
-	expandIcon            *widget.Icon
-	headerBackgroundColor color.RGBA
+	Items             []MoreItem
+	IsExpanded        bool
+	Button            *widget.Clickable
+	MoreIcon          IconButton
```

Comment:
I don't think a more icon needs to be added to collapsible. Rather than do that, you can create another struct with a name like CollapsibleWithMenu. Similar to how we have buttons and IconButton.

Reply:
The implementation of other pages where not affected. The collapsible widget was just enhanced. And since the drop down option was the major enhancement, i really dont see the need for repeating the codes since it will be similar and the only difference is the if-statement.

Reply:
One of the reasons you have up to 20 files with changes is because you changed the call to the collapsible layout method on a couple of pages. Creating a different struct removes the need for those changes. And you won't be duplicating code, you'd simply call the normal collapsible code from the widget file.

Reply:
Alright will do accordingly.